### PR TITLE
Add blur background to overlay-container

### PR DESCRIPTION
**Pull request by DanielRakh**
I haven't tested this on Android. But fuck Android amiright??? 😜

### CODE REVIEW

File: css/components/_overlay.sass [hunk 1]
```diff
@@ -6,10 +6,13 @@
   width: 100%
   height: 100%
   background-color: rgba(0, 0, 0, 0.8)
+  -webkit-backdrop-filter: blur(4px)
+  backdrop-filter: blur(4px)
   text-align: center
   color: white
   letter-spacing: 0.5px
 
+
```
Comment (DeProdigy) [inline css/components/_overlay.sass]: extra line

**Comment by DeProdigy** [comment]
@DanielRakh does this work for you?

Can you add a screenshot?

**Comment by DeProdigy** [comment]
I just tested this on iOS and it works. However it doesn't work in the regular browser hence it's been a confusion all along.

**Comment by DeProdigy** [comment]
iOS:

image


Browser in iPhone 6 mode:

image

**Comment by DanielRakh** [comment]
@DeProdigy another reason to use Safari 
screen shot 2016-08-13 at 4 15 39 pm

**Comment by DeProdigy** [comment]
Chrome....

**Comment by DeProdigy** [comment]
Alright! Glad we figured this out.

Remove the extra line and amend your comment and I'll be happy to merge :shipit:

**Comment by DanielRakh** [comment]
Done.

**Comment by DeProdigy** [comment]
@DanielRakh hah, not 2 commits. You could've just amended the original one.

Can you squash them into one commit?

**Comment by DeProdigy** [comment]
And btw if you clone the repo directly, since you are a contributor on this project, you can just branch out in the future and do a PR from that branch instead of cloning your own version of the project doing master to master PR 
image
This way you will also keep your local version of this project more in sync with the one on `Github`.

**Comment by DanielRakh** [comment]
Second commit squashed.

**Comment by DeProdigy** [comment]
@DanielRakh Nice! Merging.

**Pull request merged**